lib/logdna.rb

```diff
@@ -49,6 +49,7 @@ def initialize(key, opts={})
       begin
         request = Net::HTTP::Post.new(uri.request_uri, 'Content-Type' => 'application/json')
         request.basic_auth 'username', key
+        request['user-agent'] = opts[:rails_logdna_v] || "ruby/#{LogDNA::VERSION}"
```

Comment:
Question - this is Ruby, not necessarily Rails right? Why is there a reference to Rails?

Reply:
@dchai76, because `Rails` is using `Ruby` for logging and initializes the `Ruby`'s `Logger` with the given options. @vilyapilya is gonna put `rails/<version>` into `opts[:rails_logdna_v]` for creating logging instance for logging `Rails`

Reply:
That doesn't make sense... I can get Rails using Ruby so the dependency in that direction I understand. I don't get Ruby having a back-dependency on Rails.

Reply:
In terms of naming, that might not seem right but basically doing what's done [here](https://github.com/logdna/nodejs/blob/master/lib/logger.js#L144)

Reply:
@dchai76 I am not sure what you mean the decency in Ruby.  The header is just provides the info whether the request was sent directly from the code library or it went through some previous stages: rails.

Reply:
@vilyapilya, can you use `:user-agent` instead of `:rails_logdna_v` because this `ruby` library can be used by multiple `rails`-like higher level libraries? Thanks in advance!

Reply:
I can't use :user-agent bc Ruby does not like "-" in the name of a symbol. But I can change it to be a string.